### ISSUE TITLE
Confusing error if random walk prior scale is a variable

### ISSUE THREAD

**Issue by ChrisHIV** [ImperialCollegeLondon/epidemia#55]
If the `prior_scale` argument of `rw()` inside an `epirt()` call is a variable instead an explicitly passed numerical value, e.g. `my_guess_for_walk_step_size <- 0.2; epirt(formula = R(group, date) ~ 1 + rw(time = week, prior_scale = my_guess_for_walk_step_size))`, this gives the following opaque error:
```
Warning in parse_term(trm, data) : NAs introduced by coercion
Error in FUN(X[[i]], ...) : 
  Stan does not support NA (in ac_prior_scales) in data
failed to preprocess the data; sampling not done
Stan model 'epidemia_base' does not contain samples.
Error in stanmat[, x, drop = FALSE] : incorrect number of dimensions
```
I assume that allowing this might be tricky given the parsing of a formula, but the error could perhaps be caught and a clearer message printed (or failing that, a warning for this be given at that part of the docs relating to this).

**Comment by jscott6** [comment]
This seems to be caused by the way we are using non-standard evaluation. Definitely shouldn't cause an error, so will try and fix. Thanks!

**Comment by jscott6** [comment]
Fixed with v0.7.0
